src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1269,9 +1269,13 @@ CHIP_ERROR CASESession::ConstructTBS3Data(const uint8_t * responderOpCert, uint3
 
 CHIP_ERROR CASESession::ComputeIPK(const uint16_t sessionID, uint8_t * ipk, size_t ipkLen)
 {
+    uint8_t sid[2];
+    Encoding::LittleEndian::BufferWriter bbuf(sid, sizeof(sid));
+    bbuf.Put16(sessionID);
```

Comment:
please check bbuf status code for errors and return it if it  failed to serialize.

Reply:
done